Center header

aceburgess opened the pull request:
@afeld @orta 

I worked on fixing a different issue earlier for using HTML 5 semantic tags, and then worked on centering the header (only because I thought it may look nice). I was using the HTML 5 semantic tag, but because that's in a separate pull request which hasn't been merged I had to go back on this branch and change the css to reference the class name `.navbar` instead of the tag name `nav`.

aceburgess committed:
Center header in css for the layout template

aceburgess committed:
Fix css tag for center to class name instead of HTML5 tag name

aceburgess committed:
Fix merge conflict

aceburgess commented:
By header I now realize I meant navbar.

afeld commented:
Other pull request (#114) got merged, so feel free to update this one.

``` bash
git checkout center-header
git fetch origin
git merge origin/gh-pages
```

Also, mind including a screenshot with the new look?

aceburgess committed:
Merge branch 'gh-pages' of https://github.com/afeld/hackerhours.org into center-header

aceburgess commented:
Sure just merged the gh-pages branch with mine and commited and pushed. Here are the screenshots for the change to the navbar at the top.

![screen shot 2015-04-27 at 12 23 34 am](https://cloud.githubusercontent.com/assets/10459653/7341666/0ba45a5e-ec74-11e4-852d-35472bb0e2e7.png)
![screen shot 2015-04-27 at 12 24 36 am](https://cloud.githubusercontent.com/assets/10459653/7341667/0ba8b2ac-ec74-11e4-8f7d-a0106eb4fc32.png)

afeld commented:
Just looking at the changes to the CSS again, I'm a bit confused...it shouldn't take that many new styles to center the nav bar. Is there more to it than that? Also, can we move what's necessary to the 'base' or an even more specific 'nav' file?

aceburgess commented:
Yes I'll work on this. I added the .navbar selector in the style.scss file, but didn't touch the base.scss file. I only made one new style, but as I'm new to this, not sure what you are seeing, and if something else has changed that I have not noticed. The first commit I made (2fac62f) shows the only thing I changed. Then I fixed the merge conflict and changed the css selector back to the original class name because the other pull request with the HTML 5 tags hadn't been merged yet. Sorry still learning to code, and haven't done much HTML or CSS yet.
